net47/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Razorpay")]
+[assembly: AssemblyDescription("Razorpay SDK for .Net")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Razorpay Software Private Limited")]
+[assembly: AssemblyProduct("Razorpay.API")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```

Comment:
CopyRight should have current year also right?

Reply:
changed to `2023`

Reply:
Should it be 2015-2023?

Reply:
Also added in `nuspec` file